buildtest/cli/report.py

```diff
@@ -570,8 +570,11 @@ def print_report(self, terse=None, noheader=None, title=None, count=None):
         join_list = []
         title = title or f"Report File: {self.reportfile()}"
         table = Table(title=title, show_lines=True, expand=True)
+        selectedStyle = "green"
+        if self.display_table["state"][0] == "FAIL":
+            selectedStyle = "red"
```

Comment:
can you please add an argument to specify color where you can programatically pass a color name. Set this as optional argument in method definition such as `color = None`.